apps/api/package.json

```diff
@@ -115,7 +115,7 @@
     "swagger-ui-express": "^5.0.1",
     "uuid": "^9.0.0",
     "viem": "^2.37.8",
-    "zod": "3.25.17"
+    "zod": "3.25.76"
```

Comment:
Post tge, we should upgrade to zod 4 offically. We're already using  `zod/v4` in most places.